src/components/TeamSection/Team.tsx

```diff
@@ -16,7 +16,7 @@ const Team = (props: TeamProps) => {
       {props.teamMembers.map((member: any) => {
```

Comment:
> Closes #491
> 
> Wasn't re-rendering proprly because we are using 0 based index in both executiveTeam and projectManagers in src/components/TeamSection
> 
> We can use the name as the key or we can start projectManagers with id 22. WDYT? @Niyousha99

While this approach is a lot better, it's not full proof against idential names. What about assigning the key here when we map over `teamMembers`?

Also can you replace `any` while you're here?

Reply:
Yeah and we techncially aren't allowed to use strings as a key in React. 

I think the best option is to start projectManagers with id 22 because regardless of where we set the key, if it's the same then it won't re-render properly and it'll cause the glitching to happen again

Reply:
Sounds good, can we map over the 2 arrays for execs and PMs in teamInformation.ts and assign the key based on the length, just so we don't have to manually change all the ids every time we recruit new members (which is frequent and random throughout the year). Something like:

```
executiveTeam.map((member, index) => ({
  ...member,
  id: index + 1
}));

projectManagers.map((member, index) => ({
  ...member,
  id: teamMembers.length + 1 // Adding 1 to the length of the executiveTeam array to start at the right index
}));

```